DESCRIPTION

```diff
@@ -19,11 +19,14 @@ Encoding: UTF-8
 LazyData: true
 biocViews: Software, Infrastructure
 Imports:
-    Biostrings
+    Biostrings,
+    cluster,
+    VennDiagram,
+    igraph
 Suggests:
     testthat,
     BiocStyle,
     knitr,
     rmarkdown
-RoxygenNote: 6.1.1
+RoxygenNote: 6.1.1.9000
```

Comment:
Not sure what this is, but should probably keep consistent with the original value